Assignment2/bgDani-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,51 @@
+{
+    "name":"Daniel Berrocal Gonzalez",
+    "hometown":"Parla",
+    "age":"22",
+    "alma mater":"Universidad Politecnica de Madrid",
+    "parents name":"Miguel y Teresa",
+    "Asignaturas":{
+        "name1":"Semantic Web, Linked Data and Knowledge Graphs",
+        "name2":"Ingenieria del Software 2",
+        "teacher1":"Oscar Corcho",
+        "techer2":"Tomas San Feliu",
+        "description1":"Proporcionar la base teorica y practica sobre los fundamentos implicados en la generacion y publicacion de vocabularios y datos enlazados en la Web",
+        "description2":"Aplicar los aspectos de gestion al desarrollo software."
+    }
+}
+
+{
+    "@context":{
+        "desc":"http://www.example2.es/personaldescription/",
+        "rdfs":"http://www.w3.org/2000/01/rdf-schema#",
+        "sub":"http://www.example2.es/personaldescription/asignaturas#",
+        "name":"rdfs:label",
+        "hometown":"rdfs:label",
+        "age":"rdfs:label",
+        "alma":"rdfs:label",
+        "parents":"rdfs:label",
+        "name1":"rdfs:label",
+        "name2":"rdfs:label",
+        "teacher1":"rdfs:label",
+        "teacher2":"rdfs:label",
+        "description1":"rdfs:label",
+        "description2":"rdfs:label",
```

Comment:
You have to define each attribute with a specific vocabulary attribute name because when you want to retrieve, for example your age, if you query "rdfs:label" you will not be able to distinguish what data you ask for.